pom.xml

```diff
@@ -1405,24 +1371,8 @@
             <groupId>com.sun.jersey.contribs</groupId>
             <artifactId>*</artifactId>
           </exclusion>
-          <exclusion>
-            <groupId>org.apache.hadoop</groupId>
-            <artifactId>hadoop-yarn-server-resourcemanager</artifactId>
-          </exclusion>
         </exclusions>
       </dependency>
-      <!--
```

Comment:
Ahh .. this is going to be complicated I guess. In the fork in the company I work for, we still keep this change (SPARK-33104) alone. We did not port back SPARK-33212 yet. This is from SBT upgrade actually. 

It broke the test with Hadoop 2 so we still needed this change (SPARK-33104). Let's revert it together first as you proposed for now. I will take a look again to fix it in.

Reply:
@dongjoon-hyun feel free to revert this too if that's easier for you. I don't mind taking a look again. I will leave it to you.